rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -319,9 +319,90 @@ paths:
                     items:
                       $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/General/ForbiddenEntity.yml#/ForbiddenError"
 
-# ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-# AUTOMATED RECORDING REQUESTS
-# ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+  /user/{user_id}/recordings_quota:
+    get:
+      description: |
+        * **Available since:** 2020-1 (BPLAT-12326)
+      tags:
+        - Recordings
+      summary: Fetch used and total available recordings quota in seconds
+      operationId: userRecordingsQuota
+      parameters:
+        - in: path
+          name: user_id
+          required: true
+          description: The identity of the user to which the quota belongs to.
+          schema:
+            type: string
+      responses:
+        200:
+          description: "Successful response"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - data
+                properties:
+                  data:
+                    type: object
+                    required:
+                      - type
+                      - id
+                    properties:
+                      type:
+                        type: string
+                        example: UserRecordingsQuota
+                      id:
+                        type: string
+                        description: "The id of the entity being referenced"
+                        example: "42"
+                      attributes:
+                        type: object
+                        properties:
+                          used_quota:
+                            type: integer
+                            description: "Total duration of quota used in seconds"
+                            example: 100
+                          total_available_quota:
```

Comment:
I have a comment about this in the other PR, but let's try to use clear language here. I'd expect this API to give me two numbers: "used" and "total". If we provide "used" and "available" then we'd have to do potentially inaccurate calculations to figure out what the total amount is. Let's provide "raw" values that require as few calculations on the backend as possible, and let them do subtractions, divisions or whatever they want in the frontend.